payment/controllers.go

```diff
@@ -20,6 +20,18 @@ func Router(service *Service) chi.Router {
 
 	r.Method("POST", "/{orderID}/transactions", middleware.InstrumentHandler("CreateTransaction", CreateTransaction(service)))
 
+	r.Method("POST", "/{orderID}/transactions/anonymouscard", middleware.InstrumentHandler("CreateAnonCardTransaction", CreateAnonCardTransaction(service)))
```

Comment:
maybe use camelCase for the endpoint so it's easier to read?

```suggestion
	r.Method("POST", "/{orderID}/transactions/anonymousCard", middleware.InstrumentHandler("CreateAnonCardTransaction", CreateAnonCardTransaction(service)))
```